src/lucky_flow.cr

```diff
@@ -136,6 +136,14 @@ class LuckyFlow
     URI.parse(url).path
   end
 
+  def accept_alert
```

Comment:
What's the difference between these two methods? Since this isn't a `confirm()` modal, what are you "accept"ing?

Reply:
I think since it's an alert they do the same thing. If it was a confirm then it would do different things

Reply:
ah. Ok